dashboard/README.md

```diff
@@ -4,53 +4,49 @@
 
 Pbench Dashboard is a web-based platform for consuming indexed performance benchmark data. The platform provides a consolidated view of benchmark data within tables, charts, and other powerful visualizations. Users are able to quickly navigate through benchmark data and tune analytics through comparison tools for in-depth analysis.
 
-  
+# Dashboard directory structure
 
-## Scaffolding
+## public
 
-  
+Contains the root application `index.html` and React template artifacts.
 
-```bash
- .
- ├── public									# HTML entry
- ├── server 								# endpoint configuration
- └── src
-     ├── actions							# redux actions
-     ├── assets
-     │   ├── constants						# constants used in code
-     │   ├── images							# images
-     │   └── logo							# Pbench logos
-     ├── modules
-     │   ├── components						# functional components
-     │   └── containers						# parent components
-     ├── reducers							# redux reducer
-     ├── store								# redux store
-     └── utils								# utility functions
-```
+(BTW, do we really want/need to retain the React logos here? We don't use them, do
+we?)
```

Comment:
If we don't use them, I don't see any point in retaining them.

I'm not attuned to the Open-Source-ness here..._should_ we be making use of them?

Reply:
Removed those logos

Reply:
Please be sure this change is recorded in the commit message (typically, we should update the description with the final text of the commit that would be used in the squashed commit message).